hw/ip/pwm/doc/checklist.md

```diff
@@ -16,15 +16,15 @@ All checklist items refer to the content in the [Checklist.]({{< relref "/doc/pr
 
 Type          | Item                           | Resolution  | Note/Collaterals
 --------------|--------------------------------|-------------|------------------
-Documentation | [SPEC_COMPLETE][]              | Not Started | [PWM Design Spec]({{<relref "hw/ip/pwm/doc" >}})
-Documentation | [CSR_DEFINED][]                | Not Started |
-RTL           | [CLKRST_CONNECTED][]           | Not Started |
-RTL           | [IP_TOP][]                     | Not Started |
-RTL           | [IP_INSTANTIABLE][]            | Not Started |
-RTL           | [PHYSICAL_MACROS_DEFINED_80][] | Not Started |
-RTL           | [FUNC_IMPLEMENTED][]           | Not Started |
-RTL           | [ASSERT_KNOWN_ADDED][]         | Not Started |
-Code Quality  | [LINT_SETUP][]                 | Not Started |
+Documentation | [SPEC_COMPLETE][]              | Done        | [PWM Design Spec]({{<relref "hw/ip/pwm/doc" >}})
```

Comment:
```suggestion
Documentation | [SPEC_COMPLETE][]              | Done        | [PWM Design Spec]({{<relref "." >}})
```

Let's please avoid absolute links to make IP blocks easier to relocate.

Reply:
Ah, I didn't know relref worked this way.  I thought they were all relative the project root.  Sure, no problem.